main.cpp

```diff
@@ -432,6 +432,27 @@ Thing 7) Entertainment
     3) work for long haul
  */
 
+struct Entertainment
+{
+    // name of act
+    std::string nameOfAct = "WOW";
+    // cost of hire
+    double costOfHire = 2359.23;
+    // allocated time
+    std::string allocatedTime = "2pm - 5pm everyday";
+    // number of performers
+    int numOfPerformers = 7;
+    // location of performance
+    std::string locationOfPerformance = "Sky Deck";
+
+    // do acrobats
+    bool doAcrobats(bool safetyEquipmentIsPresent); //returns if they are able to perform acrobats.
+    // family friendly
+    void familyFriendly();
```

Comment:
These function names don't match the descriptions
Function names should describe the actions they perform. 
They should have verbs in their name
- [ ] revise